themes/my-theme/scripts/article/add-classes.js

```diff
@@ -0,0 +1,5 @@
+hexo.extend.tag.register('add_classes', function (args) {
```

Comment:
I think the tag name is misleading.
It does not mention it creates a new paragraph.
Maybe "p_with_classes"?